core/src/components/input/test/bottom-content/input.e2e.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('input: bottom content', () => {
+  test.beforeEach(({ skip }) => {
+    skip.rtl();
+    skip.mode('ios', 'Rendering is the same across modes');
+  });
+  test('should not render bottom content if no hint or counter is enabled', async ({ page }) => {
+    await page.setContent(`<ion-input label="my input"></ion-input>`);
+
+    const bottomEl = page.locator('ion-input .item-bottom');
+    await expect(bottomEl).toHaveCount(0);
+  });
+});
+
+test.describe('input: hint text', () => {
+  test.describe('input: hint text functionality', () => {
```

Comment:
I feel like these screenshot tests should include the whole input, not just the bottom part, to check things like position and padding between the two.

Reply:
The padding for the bottom text is set on the `.input-bottom` element which is captured in the screenshots. I can also add some screenshots to the `basic` test which checks the component as a whole. Would that work?

Reply:
Yeah, that sounds good 👍

Reply:
I'm going to add this in a separate PR with the `basic` tests that look at the component as a whole. Should be in https://github.com/ionic-team/ionic-framework/pull/26238 once I get everything ready.